assets/stylesheets/host.css

```diff
@@ -9,4 +10,231 @@ body {
+  padding-bottom: 15px;
+  width: 1000px;
+  height: 550px;
+  border: 5px solid;
+  border-radius: 20px;
+}
+
+@keyframes modalFade {
+  0% {
+   opacity: 0%;
+  }
+  100% {
+    opacity: 100%;
+   }
+}
+
+.modal-text {
+  width: 950px;
+  height: 450px;
+  margin: 25px 25px 0px 25px;
+  overflow: scroll;
+}
+
+.modal-activity-title {
+  margin-top: 0px;
+  margin-bottom: 0px;
+  font-size: 48px;
+  font-weight: bold;
+}
+
+.activity-text-full {
+  line-height: 1.5;
+  padding-right: 25px;
 }
+
+.modal-delineation {
+  margin: 15px 0px 15px 0px;
+  border: 1px solid black;
+  opacity: 70%;
+}
+
+.close-modal-btn {
+  background-color: #ed9393;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 100px;
+  height: 40px;
+  font-size: 18px;
+  border-radius: 8px;
+}
+
+.close-modal-btn:hover {
+  filter: brightness(0.9);
+}
+
+.close-modal-btn:active {
+  filter: brightness(0.7);
+  box-shadow: 3px 3px rgba(0, 0, 0, 0.8);
+}
+
+.host-header {
+  font-size: 48px;
+  font-weight: bold;
+  margin-bottom: -5px;
+}
+
+.activity-selection {
+  background-color: white;
+  display: flex;
+  flex-direction: row;
+  border: 2px solid;
+  height: 400px;
+  border-radius: 35px;
+  margin-top: 15px;
+  padding: 25px;
+}
+
+.activity-description {
+  width: 40%;
+  font-size: 18px;
+  margin-right: 30px;
+}
+
+.library-model-container {
+  position: relative;
+}
+
+.library-model-container:after {
+  background-color: #aed8f2;
+  content: "\25bc";
+  color: white;
+  right: 0vw;
+  position: absolute;
+  height: 20px;
+  padding: 4px 10px 4px 10px;
+  border: 1px solid gray;
+  border-radius: 0px 8px 8px 0px;
+  pointer-events: none;
+}
+
+.library-model-container:focus-within::after {
+  height: 18px;
+  padding: 4px 10px 4px 10px;
+  border: 2px solid #085ccc;
+  border-left: 1px solid gray;
+  border-radius: 0px 8px 8px 0px;
+}
+
+.library-model-container:hover {
+  filter: brightness(0.9);
+}
+
+.library-model {
+  width: 100%;
+  height: 30px;
+  font-size: 18px;
+  padding-left: 5px;
+  border-radius: 8px;
+}
+
+.activity-text-short {
+  max-height: 200px;
+}
+
+.read-more-btn {
+  background-color: #aed8f2;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 150px;
+  height: 50px;
+  font-size: 18px;
+  border-radius: 8px;
+}
+
+.read-more-btn:hover {
+  filter: brightness(0.9);
+}
+
+.read-more-btn:active {
+  filter: brightness(0.7);
+  box-shadow: 3px 3px rgba(0, 0, 0, 0.8);
+}
+
+.img-wrapper {
+  width: 60%;
+}
+
+.img {
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+}
+
+.form-input-wrapper {
+  display: flex;
+  align-items: flex-end;
+  justify-content: center;
+  margin-top: 30px;
+}
+
+.form-label {
+  display: block;
+  font-size: 20px;
+  margin-bottom: 5px;
+}
+
+.form-input {
+  display: block;
+  width: 250px;
+  height: 30px;
+}
+
+.center-input {
+  margin-left: 100px;
+  margin-right: 100px;
+}
+
+.launch-btn {
+  background-color: #bfebbe;
+  /* margin: 20px 0px 0px 0px; */
```

Comment:
Please remove commented-out code.

Reply:
Gah - that's embarrassing 🙈; totally rookie gaffe on my part! I definitely did not mean to have that in there & must've glossed over it when I was pushing code to this branch.